tools/msvs/msi/product.wxs

```diff
@@ -110,18 +110,18 @@
 
     <Directory Id="TARGETDIR" Name="SourceDir">
       <Directory Id="ProgramMenuFolder">
-        <Directory Id="ApplicationProgramsFolder" Name="io.js"/>
+        <Directory Id="ApplicationProgramsFolder" Name="Node.js"/>
       </Directory>
 
       <Directory Id="$(var.ProgramFilesFolderId)">
-        <Directory Id="INSTALLDIR" Name="iojs">
+        <Directory Id="INSTALLDIR" Name="node">
```

Comment:
This was `nodejs` in joyent/node https://github.com/joyent/node/blob/v0.12/tools/msvs/msi/product.wxs#L119 . Shouldn't we keep consistency with that?